src/lesson4/ArrayTask.java

```diff
@@ -0,0 +1,45 @@
+package lesson4;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class ArrayTask {
+    Scanner sc = new Scanner(System.in);
+    private String[] word1;
+    private String[] word2;
+
+    public void getWord() {
+        word1 = sc.nextLine().split(" ");
+//        System.out.println("Enter the second word array: ");
+        word2 = sc.nextLine().split(" ");
+        System.out.print("1st -> " + Arrays.toString(word1) + "\t");
+        System.out.println("2nd -> " + Arrays.toString(word2));
+    }
+
+    public String complyWord(String[] word) {
+        StringBuilder compliedWord = new StringBuilder();
+        for (String s : word) {
+            compliedWord.append(s);
+        }
+        return compliedWord.toString();
+    }
+
+    public boolean areEqual(String stringWord1, String stringWord2) {
+        return stringWord1.equals(stringWord2);
+    }
+
+    public void launch() {
+        System.out.println("Enter two strings to compare (use space to split the string):");
+//        System.out.println("Enter the first word array: ");  doesn't work after first loop, how to improve?
```

Comment:
do {} while  loop can help

Reply:
but then the 34 line doesn't work properly, such expression makes it impossible to quit during the first loop

Reply:
you can make addition check for exit command in the loop,  and if command is exit than do  return, and also do while {} can be with true statement,